mreg_cli/api/fields.py

```diff
@@ -20,7 +20,7 @@ class MACAddressField(FrozenModel):
 
     address: str
 
-    @field_validator("address", mode="before")
+    @field_validator("address", mode="after")
```

Comment:
By using `"after"`, we let Pydantic ensure the value is a string before we proceed with validating and manipulating it.